lib/parser/classes/MusicResponsiveListItem.ts

```diff
@@ -0,0 +1,175 @@
+// TODO: this needs a refactor
+// seems like a mess to use
```

Comment:
YouTube Music uses 1 single renderer for all kinds of items, and it's a bit inconsistent. Agreed, nonetheless.